tests/x11/x3270_ssl.pm

```diff
@@ -64,7 +71,12 @@ sub run {
     mouse_hide(1);
 
     #Launch x3270
-    type_string "x3270 -trace -tracefile $tracelog_file L:localhost:8443\n";
+    if ($current_ver < 3.6) {
```

Comment:
Maybe update code as below is better:
my $noverifycert = ($current_ver < 3.6 ?  '' : '-noverifycert');
type_string "x3270 -trace $noverifycert -tracefile $tracelog_file L:localhost:8443\n";

Reply:
Thanks for the comment, I will update it and retest it in VR. :)

Reply:
Done.